install/install.sh

```
@@ -0,0 +1,184 @@
+        echo "------------------------------------------------------------"
+        echo -e "\e[31m❌ Clone failed for repository: $repo_crons\e[0m"
+        return 1
+    fi
+}
+
+
+loveletter_service(){
+	echo -e "\nInstalling Cron Jobs Services ..."
+	clean_dir "$repo_crons"
+	clone_repo $tag $repo_crons
+	sudo cp $(pwd)/Rpi_Crons/loveletter.service /etc/systemd/system/
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)